### PR TITLE
Increase limits FCash

### PR DESCRIPTION
**Pull request by brucedonovan**
Increase fCash limits ( max and dust) for usdc/fusdc and dai/fdai.

### CODE REVIEW

File: YPP-0103.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+FDAI2303 dec set: 18
+FDAI2306 max set: 1000000
+FDAI2306 min set: 1000
+FDAI2306 dec set: 18
+FDAI2303 max set: 1000000
+FDAI2303 min set: 1000
+FDAI2303 dec set: 6
+FDAI2306 max set: 1000000
+FDAI2306 min set: 1000
+FDAI2306 dec set: 6
```
Comment (alcueca) [inline YPP-0103.md:195]: The tests are not checking FUSDC2303 or FUSDC2306

**Review by alcueca — CHANGES_REQUESTED** [review]
I think that we might not be applying the new limits where we want them.

File: YPP-0103.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+/// @param base identifier (bytes6 tag)
+/// @param ilk identifier (bytes6 tag)
+/// @param Maximum global debt allowed for this pair, modified by decimals
+/// @param Minimum vault debt allowed for this pair, modified by decimals
+/// @param Decimals to append to global and vault debt.
+export const newLimits: Array<[string, string, number, number, number]> = [
+  [DAI, FDAI2303, 1000000, 1000, 18],
+  [DAI, FDAI2306, 1000000, 1000, 18],
+  [USDC, FDAI2303, 1000000, 1000, 6],
+  [USDC, FDAI2306, 1000000, 1000, 6],
```
Comment (alcueca) [inline YPP-0103.md:32]: This doesn't reflect the [actual config](https://github.com/yieldprotocol/environments-v2/blob/b3a1edf99d8f30feaae74e48168d37d3e6444c99/scripts/governance/update/updateLimits/updateLimits.mainnet.config.ts#L39), where USDC relates to FUSDC ilks.

Comment (brucedonovan) [reply YPP-0103.md:32]: fixed

**Review by alcueca — APPROVED** [review]

**Comment by iamsahu** [comment]
Proposed: `0x51f5121988d98da858cd71e521218031a03381b85f01ae5d6a0a96c7dec3d188`

[Transaction](https://etherscan.io/tx/0x1d492cc8a8e86028412ee1d800ed2b94c87da543f3b8db9f78b8b428c820f2a1)
[Tenderly fork](https://dashboard.tenderly.co/Yield/v2/fork/292db56c-55d0-49f2-aec0-0c4cc4de7d9f)

**Comment by iamsahu** [comment]
[Executed](https://etherscan.io/tx/0xcbc89d16701d211f3633fc4cd9f7e23be161fccd4fe35ee62f2a9c97953ae64d)

**Pull request merged**